src/ipc/reboot.ts

```diff
@@ -0,0 +1,13 @@
+import { IpcMain, IpcMainInvokeEvent } from 'electron'
+import exec from '../utils/exec'
+
+export const channel = 'reboot'
+
+/**
+ * Reboot the machine
+ */
+export default function register(ipcMain: IpcMain): void {
+  ipcMain.handle(channel, async (event: IpcMainInvokeEvent) => {
+    await exec('reboot')
```

Comment:
So I think this will only work on non-locked down machines. If you want it to work in both, use `systemctl -i reboot`

Reply:
+1 to checking what works on a locked down machine (and we do want it to work.)

If @umbernhard's suggestion works, great. Otherwise, we may have to `sudo reboot` and grant `vx-ui` user the right to sudo for that command.